testing/e2e/config/config.go

```diff
@@ -156,12 +157,70 @@ type AdditionalService struct {
 	Replicas int `json:"replicas"`
 }
 
+// NethermindConfig holds specific configuration for Nethermind client
+type NethermindConfig struct {
+	// Specific settings for Nethermind
+	SyncConfig struct {
+		FastSync                  bool `json:"FastSync"`
+		DownloadBodiesInFastSync bool `json:"DownloadBodiesInFastSync"`
+		DownloadReceiptsInFastSync bool `json:"DownloadReceiptsInFastSync"`
+	} `json:"SyncConfig"`
+	
+	// Settings to improve finalization
+	ConsensusConfig struct {
+		ForceSealing bool `json:"ForceSealing"`
+		TargetBlockGasLimit int64 `json:"TargetBlockGasLimit"`
+	} `json:"ConsensusConfig"`
```

Comment:
"TargetBlockGasLimit": 30000000 is already been set in kurtosis, not required in default it is redundant here.
 
If you still want to customize it, pls see the correct usage it is under `Block` namespace. https://docs.nethermind.io/fundamentals/configuration/#blocks-targetblockgaslimit